### ISSUE TITLE
Duo icon request

### ISSUE THREAD

**Issue by cschaaf73** [walkxcode/dashboard-icons#306]
### Type

Icon request

### Info

Duo icon request

### Icon(s)

<img width="286" alt="Duo" src="https://user-images.githubusercontent.com/18709993/222966810-92930bcc-3d61-4c6d-a2c9-62ec9031e009.png">

**Comment by spyhuntergenral** [comment]
#313

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/duo.png
https://github.com/walkxcode/dashboard-icons/blob/new-standards/svg/duo.svg
